middleware/queue/source/group/handle.cpp

```diff
@@ -243,9 +243,8 @@ namespace casual
                      Trace trace{ "queue::handle::dequeue::Request::handle"};
                      log::line( verbose::log, "message: ", message);
 
-                     auto now = platform::time::clock::type::now();
-
-                     auto reply = state.queuebase.dequeue( message, now);
+                     // send an empty reply on error TODO: is this the api we want?
```

Comment:
How does the _forward_ act on an empty reply? And is it detected as an error?

Reply:
It logs an error and rolls back the transaction, restarting the 'flow' when it gets the rollback_reply. At least that's my interpretation: https://github.com/casualcore/casual/blob/patch/1.6/main/middleware/queue/source/forward/handle.cpp#L490

Reply:
Ok